theforeman.org/pipelines/test/foreman/vars.groovy

```diff
@@ -0,0 +1,30 @@
+def foreman_assets_versions = [
+    'develop': [
+        'ruby': ['2.7']
+    ],
+    '3.3-stable': [
+        'ruby': ['2.7']
+    ],
+    '3.2-stable': [
+        'ruby': ['2.7']
+    ],
+    '3.1-stable': [
+        'ruby': ['2.7']
+    ]
+]
+
+def foreman_unit_versions = [
+    'develop': [
+        'ruby': ['2.7', '2.5']
+    ],
+    '3.3-stable': [
+        'ruby': ['2.7', '2.5']
+    ],
+    '3.2-stable': [
+        'ruby': ['2.7', '2.5']
+    ],
+    '3.1-stable': [
+        'ruby': ['2.7', '2.5']
+    ]
+]
```

Comment:
I don't think the duplication is needed. In the integration tests you already only have Ruby 2.7 as the only axis anyway.